src/main/java/org/ethereum/beacon/discovery/schema/IdentitySchemaV4Interpreter.java

```diff
@@ -116,4 +128,15 @@ public NodeRecord createWithNewAddress(
   private static InetAddress getInetAddress(final Bytes address) throws UnknownHostException {
     return InetAddress.getByAddress(address.toArrayUnsafe());
   }
+
+  private static Stream<EnrField> getAllFields(final NodeRecord nodeRecord) {
```

Comment:
nit: `streamAllFields` tends to be the convention for things that return a `Stream`.

Reply:
done.